### ISSUE TITLE
Guzzle 7 incompatibility

### ISSUE THREAD

**Issue by brazenvoid** [DocRaptor/docraptor-php#23]
Laravel 8 requires Guzzle 7 but docraptor doesn't support it.

**Comment by brazenvoid** [comment]
I tried by moving the source to main project and running with Guzzle 7 and it worked fine. 

Please update the dependency requirement.

**Comment by airdrummingfool** [comment]
I have created a PR (#24) that adds support for Guzzle 7. Until it is merged, you can use it by making the following changes to `composer.json`:

Add a vcs repository entry for my fork:
```
    "repositories": [
      {
        "type": "vcs",
        "url": "https://github.com/airdrummingfool/docraptor-php"
      }
    ],
```

switch the docraptor require to use my `guzzle-7` branch (must include `dev-` prefix):
```
"docraptor/docraptor": "dev-guzzle-7",
```

Use at your own risk! I will delete the fork after my PR is merged.

**Comment by Rodeoclash** [comment]
I've thumbed up everything I can in this PR but also feel the need to add a comment. I'd love for this to be merged!

**Comment by Rodeoclash** [comment]
It's been ~5 months since this issue was raised and a PR provided that hasn't been merged. Is it safe to assume that we shouldn't expect further updates to the DocRaptor PHP library and instead should look at interfacing with the endpoints directly?

**Comment by SteelWagstaff** [comment]
@Rodeoclash see recent response on this issue from @jason-o-matic here: https://github.com/DocRaptor/docraptor-php/pull/24#issuecomment-896316141

**Comment by christianl-das** [comment]
There is now an additional issue with the use of `build_query()`. That function no longer exists in the most recent versions of Guzzle. It has been replaced by `Build::query()`.

**Comment by esquivalient** [comment]
I released version 3.0 today. It should solve this issue.
